environments/aws/s3.tf

```diff
@@ -10,6 +10,6 @@ module "flyte_data" {
 }
 
 output "bucket" {
-  value = module.flyte_data.bucket.name
+  value = module.flyte_data.s3_bucket_id
```

Comment:
`name` here is an attribute from `bucket`. I'm not sure if `s3_bucket_id` would work. If you specified that field under the `flyte_data` module then it should work for you but no one else

Reply:
I've tested both values and:
- `bucket.name` yields an error (something like: `module 'flyte_data' does not have a 'bucket' attribute`)
- `s3_bucket_id` works

Reply:
@qchenevier right. It's my fault. For some reason the test version was different and it was using an old output name. It's fixed now. Sorry again.